api/client/client.go

```diff
@@ -0,0 +1,109 @@
+package client
+
+import (
+	"context"
+	"io"
+
+	"github.com/andres-teleport/overseer/api"
+	"github.com/andres-teleport/overseer/api/authentication"
+	"google.golang.org/grpc"
+)
+
+type Client struct {
+	client api.JobworkerServiceClient
+}
+
+type Status struct {
+	Status   string
```

Comment:
Not sure I understand, are you saying that I should use `api.Status` directly instead of creating this new type?

Reply:
yes, the `StatusResponse` gRPC message  uses enum type for status field,  for the same reason client library should also operate on enum status value instead of strings.